test/issues/general/test_9456.test

```diff
@@ -0,0 +1,54 @@
+# name: test/issues/general/test_9456.test
+# description: Issue 9456: EXISTS operator returns NULL when multiple predicates are present
```

Comment:
Do you mean "exists operator **does not** return NULL when multiple predicates are present?"

Reply:
I've copied this from the original issue, as we did for other issue test cases